api/types/types.go

```diff
@@ -175,6 +175,7 @@ type Info struct {
 	SystemTime         string
 	LoggingDriver      string
 	CgroupDriver       string
+	CgroupVersion      int `json:",omitempty"`
```

Comment:
I was wondering if it would be safer to make this a string?

Reply:
updated